aten/src/ATen/native/PointwiseOps.cpp

```diff
@@ -40,6 +48,7 @@ TORCH_META_FUNC(addcdiv)
         "The future addcdiv behavior is just the latter implementation: ",
         "(input + value * tensor1 / tensor2), for all dtypes.");
   }
+  // TODO: add support for cpu_scalar in addcdiv
```

Comment:
We'd prefer tracking issues in the GitHub repo over TODOs in the codebase so feel free to convert this TODO into a feature request issue! But let's try to minimize the TODOs in the codebase.

Reply:
That makes sense! I'll do that now.